nexuscli/acls.py

```diff
@@ -13,6 +13,9 @@
           }
 
 _PROJECTS_READ_ = "projects/read"
+_SCHEMAS_READ_ = "schemas/read"
```

Comment:
remove

Reply:
this permission doesn't exist

Reply:
why would you not want to allow users to read schemas ?